merged.graphql

```diff
@@ -24,9 +27,57 @@ type PluginReturn {
 	errors: [Status!]!
 }
 
+type PluginMasterReturn  {
+	live_site: LiveSiteConfigType
+	slack: SlackConfigType
+	twilio: TwilioConfigType
+	twitter: TwitterConfigType
+}
+
+type MessageReturn {
+    message: String
+    config: MetaDataType!
+    _id: ID!
+    createdAt: String!
+    updatedAt: String!
+}
+
+type LiveSiteConfigType {
+		title: String
+		icon: String
+	}
+
+
+type SlackConfigType {
+		channels: [String!]!
+		at_channel: Boolean!
+		at_here: Boolean!
+	}
+
+type TwilioConfigType {
+		numbers: [String!]
+    groups: [String!]
+	}
+
+type TwitterConfigType {
+		_: Boolean
+	}
+type MetaDataType {
```

Comment:
sooo does this work? Since the ts type has fields as non-required but these are required?